clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -265,4 +277,13 @@ public UUIDSerde() {
     static public Serde<Void> Void() {
         return new VoidSerde();
     }
+
+    /*
+     * A serde for {@code List} type
+     */
+    public static <L extends List, Inner> Serde<List<Inner>>
```

Comment:
Should be `<L extends List<Inner>, Inner>` to avoid raw type warning and make build pass

Reply:
Hmmm, in this case it is impossible to create a list serder or I am doing something wrong.
For ex., in my tests I used to have something like:
```
Serde<List<Integer>> listSerde = Serdes.ListSerde(ArrayList.class, Serdes.Integer());
```
It does not compile now, I've tried changing it to:
```
Serde<List<Integer>> listSerde = Serdes.ListSerde((Class<List<Integer>>)List.class, Serdes.Integer());
```

Oooofff, generics are hard 😅

Reply:
Thinking about it once more, it might not work what I suggested, because if you want to call `Serde.ListSerde(ArrayList.class, ...)` the `ArrayList` does not specify any inner type information (but is a raw type) and thus it won't compile. I guess, we need to leave it as-is, and suppress the "raw type" warning (might be worth to add a comment why the raw type warning cannot be avoided for this case).